spec/descriptions/tagSyntheticSettings.md

```diff
@@ -43,7 +43,7 @@ The API endpoints of this group can be used to manage Synthetic Locations, Synth
         - If timeout value in test configuration is not provided, the default value is **1m** for HTTPAction and HTTPScript tests. 
           BrowserScript, WebpageAction, and WebpageScript tests use the smaller one of `maxTimeout` and `testFrequency` as the actual timeout value.
     - **XXXConfiguration** The configuration corresponding to the syntheticType. Configuration types are HTTPActionConfiguration, HTTPScriptConfiguration,
-      BrowserScriptConfiguration (Beta), WebpageActionConfiguration (Beta), and WebpageScriptConfiguration (Beta). 
+      BrowserScriptConfiguration, WebpageActionConfiguration, WebpageScriptConfiguration, and SSLCertificateConfiguration (Beta). 
```

Comment:
```suggestion
      BrowserScriptConfiguration, WebpageActionConfiguration, WebpageScriptConfiguration, and SSLCertificateConfiguration. 
```

Reply:
A similar sentence about Synthetic types is also in Location properties part at this [line](https://github.com/instana/openapi/blob/b8bef1f90539cc2625845191d9839b770901f588/spec/descriptions/tagSyntheticSettings.md?plain=1#L11). It needs to be updated too.